packages/map/src/GMap.ts

```diff
@@ -564,6 +564,9 @@ export class GMap extends HTMLWidget {
                     heading,
                     pitch: 0
                 });
+                if(context.hideStreetViewMarker()){
+                    marker.setVisible(false);
+                }
```

Comment:
Toggling back to "show" will cause an issue?

Reply:
Toggling back to show will work once they call `streetViewAt` again.